lib/manual_relocator.rb

```diff
@@ -37,11 +37,11 @@ def fetch_manual(slug)
   end
 
   def old_section_ids
-    @old_section_ids ||= old_manual.editions.flat_map(&:section_ids).uniq
+    @old_section_ids ||= old_manual.editions.flat_map(&:section_uuids).uniq
   end
 
   def new_section_ids
-    @new_section_ids ||= new_manual.editions.flat_map(&:section_ids).uniq
+    @new_section_ids ||= new_manual.editions.flat_map(&:section_uuids).uniq
```

Comment:
I consciously didn't rename them in this commit. There are quite a few variables/methods in the code that still refer to `id` instead of `uuid` (when dealing with `Section`s) and I plan to open a separate PR to try to rename all those up together.